validator/client/src/main/java/tech/pegasys/teku/validator/client/Validator.java

```diff
@@ -38,7 +35,7 @@ public Signer getSigner() {
     return signer;
   }
 
-  public Optional<Bytes32> getGraffiti() {
+  public Graffiti getGraffiti() {
```

Comment:
I'd keep this signature the same (returning `Optional<Bytes32>` and just load the graffiti here.